cScripts/CavFnc/functions/mission/fn_teleportToGroup.sqf

```diff
@@ -0,0 +1,69 @@
+#include "..\script_component.hpp";
+/*
+ * This teleports the player to his group leader.
+ * If group leader is in a vehicle, player is teleported into the vehicle.
+ * Can only be used once per session
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [this] call cScripts_fnc_teleportToGroup
+ *
+ * Public: Yes
+ */
+
+params [
+    ["_object", objNull, [objNull]]
+];
+
+missionNamespace setVariable ["usedTeleport",false];
```

Comment:
might have brain melted on that.
this is for all players meaning if I use it the next dude cant.

so you can go back to what you had or append guids to the var array instead of using a book and check if player is in the array.